tee-worker/scripts/litentry/clean_worker.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+pids=$(pgrep -f "integritee-service")
```

Comment:
Would `killall integritee-service` work?

Reply:
It actually did not work, but this is no longer needed, Will be removed ✅